src/test/java/net/syscon/elite/executablespecification/steps/AuthTokenHelper.java

```diff
@@ -301,7 +301,7 @@ private String updateAlert() {
     private String courtHearingMaintainer() {
         return jwtAuthenticationHelper.createJwt(
                 JwtParameters.builder()
-                        .username("ITAG_USER")
+                        .username("DOES_NOT_EXIST")
```

Comment:
Is there a better (clearer) way of doing this?

Reply:
Depends what you're trying to do.  Currently reads to me that you want client creds with a user that does not exist?

Reply:
Username could be the clientId.  @petergphillips maybe this should be `sub` now?

Reply:
maybe, although  `username` is baked into spring security oauth2 so won't be coming out anytime soon.  I would have thought it would be better to not  pass in `username` at all, since that will be what pecs are doing.